connectors/snowflake-sink/2.10.3/snowflake-sink.md

```diff
@@ -0,0 +1,385 @@
+| `host`                     | String  | Yes      | ""(empty string)                                        | The host URL of the snowflake service.                                                                                                                                                                                                                                                                                                                         |
+| `database`                 | String  | Yes      | ""(empty string)                                        | The Snowflake database where the connector will sink data.                                                                                                                                                                                                                                                                                                     |
+| `schema`                   | String  | Yes      | ""(empty string)                                        | The Snowflake schema belongs to the level below the Snowflake database and consists of a set of tables.                                                                                                                                                                                                                                                        |
+| `tableName`                | String  | No       | ""(empty string)                                        | If the `autoCreateTable` option is set to `false`, the Snowflake connector will persist messages to this table.                                                                                                                                                                                                                                                |
+| `warehouse`                | String  | No       | ""(empty string)                                        | The warehouse name in the snowflake. By default, no warehouse name is set.                                                                                                                                                                                                                                                                                     |
+| `bufferCountRecords`       | int     | No       | 10_000                                                  | The number of records that are buffered in the memory before they are ingested to Snowflake. By default, it is set to `10_000`.                                                                                                                                                                                                                                |
```

Comment:
```suggestion
| `bufferCountRecords`       | int     | No       | 10_000                                                  | The number of records that are buffered in the memory before they are ingested by Snowflake. By default, it is set to `10_000`.                                                                                                                                                                                                                                |
```

Reply:
The idea here is to insert into the Snowflake, not insert through the Snowflake.